docs/src/overview/build-deploy.md

```diff
@@ -36,11 +36,11 @@ The live environment is composed of multiple containers&mdash;both for your appl
 
 Based on your application type the system will select one of our pre-built container images and run the following:
 
-1. First, any dependencies specified in the `.platform.app.yaml` file are installed.  Those include tools like Sass, Gulp, Drupal Console, or any others that you may need.  
+1. First, any dependencies specified in the `.platform.app.yaml` file are installed. Those include tools like Sass, Webpack, Drupal Console, or any others that you may need.
```

Comment:
Is Gulp not cool anymore? I can't keep up with JS fashion... :smile:

Reply:
Yeah modern webapps configurations don't really use grunt/gulp anymore. All major frameworks use and include documentation for Webpack.